thentos-core/src/ThentosDocs.hs

```diff
@@ -0,0 +1,13 @@
+module ThentosDocs (makeMain) where
+
+import Data.Thyme.Time ()
```

Comment:
```
$ git grep 'import Data.Thyme.Time ()' | grep required
thentos-adhocracy/src/Thentos/Adhocracy3/Types.hs:import Data.Thyme.Time () -- required for NominalDiffTime's num instance
thentos-core/src/Thentos/Types.hs:import Data.Thyme.Time () -- required for NominalDiffTime's num instance
```

It would be nice to be able to import instances explicitly as well...